### PR TITLE
Feature(Service): upgrade packages + add tests

### PR DESCRIPTION
**Pull request by dobobaie**

### CODE REVIEW

**Comment by jonathanasquier** [comment]
Now that you have a test suite could you set up the CI please?

**Comment by dobobaie** [comment]
> Now that you have a test suite could you set up the CI please?

Sure !

**Comment by jonathanasquier** [comment]
done

**Comment by jonathanasquier** [comment]
@dobobaie up and ping me when CI passes

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
```
Comment (jonathanasquier) [inline .circleci/config.yml]: please use "real" node image

**Comment by dobobaie** [comment]
@jonathanasquier 👍

**Pull request merged**